src/gtkchart.c

```diff
@@ -62,8 +62,35 @@ struct _GtkChartClass
   GtkWidgetClass parent_class;
 };
 
+// Colors
+GdkRGBA bg_color, white, blue, red, line, grid;
+
+// Font
+gchar *font_name;
+
 G_DEFINE_TYPE (GtkChart, gtk_chart, GTK_TYPE_WIDGET)
 
+void
+gtk_chart_get_theme_colors (GtkWidget *parent)
+{
+  GtkStyleContext *context = gtk_widget_get_style_context (parent);
+  gtk_style_context_lookup_color (context, "theme_bg_color", &bg_color);
+  gtk_style_context_lookup_color (context, "theme_fg_color", &white);
+  gtk_style_context_lookup_color (context, "theme_selected_bg_color", &blue);
+  gtk_style_context_lookup_color (context, "error_color", &red);
+  gtk_style_context_lookup_color (context, "theme_selected_bg_color", &line);
+  gtk_style_context_lookup_color (context, "theme_fg_color", &grid);
+  grid.alpha = 0.1;
+}
+
```

Comment:
Since we are now using colors from the system theme it does not make much sense to use variable names such as white, blue, and red.

Perhaps rename variable 'white' to 'text' and simply delete 'blue' and 'red' color variables because they seem unused. What do you think?

Reply:
I was thinking about it, but I didn't wanted to edit too much the code. But yes, it makes sense.